app.js

```diff
@@ -49,13 +49,10 @@ app.use("/static/img/logos", express.static("public/img/logos"));
 app.use(helmet());
 if (config.get("server.useRedis")) {
   // Messy but avoids any open file handles.
-  const redisClient =
-    createClient({
-      legacyMode: true,
-      password: config.get(process.env.NODE_ENV + ".redisPass"),
-    });
-
-  //const redisStore = require("connect-redis")(session);
+  const redisClient = createClient({
+    legacyMode: true,
```

Comment:
Does this prevent the issue with supplying a password when no password is needed? I would like to make sure the redis instance is also secure with a password.